src/components/pages/ActionListPage.tsx

```diff
@@ -462,10 +462,10 @@ function ActionListPage({ page }: ActionListPageProps) {
               </Select>
             </FormControl>
           </Box>
-        </Container>
+        </MUIContainer>
       </ViewSelectorBar>
 
-      <Container fluid="lg" className="mb-5">
+      <MUIContainer fixed maxWidth="lg" sx={{ mb: 5 }}>
```

Comment:
Let's use xl as maximum width

```suggestion
      <MUIContainer fixed maxWidth="xl" sx={{ mb: 5 }}>
```